data/alignments/eng/LEB/SBLGNT-LEB-manual.toml

```diff
@@ -0,0 +1,33 @@
+[source]
+
+identifier = "SBLGNT"
+copyright = "Copyright 2010 by the Society of Biblical Literature and Logos Bible Software."
+license = "CC-BY-4.0"
+
+[source.metadata]
+copyright = "Copyright (c) 2023 by Clear Bible, Inc."
+license = "CC-BY 4.0"
+
+[target]
+
+identifier = "LEB"
+copyright = "Copyright © 2012 Logos Bible Software"
+license = "Custom"
+licensenotes = "'You can give away the Lexham English Bible, but you can't sell it on its own.'"
+name.eng = "Lexham English Bible"
+url = "http://www.lexhamenglishbible.com"
+
+[target.metadata]
+copyright = "Copyright (c) 2023 by Clear Bible, Inc."
+license = "CC-BY 4.0"
+
+[alignment]
+
+identifier = "NA27-LEB-manual"
```

Comment:
@RickBrannan  By convention, this `identifier` should match the alignment filename, in this case `SBLGNT-LEB-manual`.

Reply:
Hm, I thought I changed that with the second commit. Will look again.